examples/dodge-the-creeps/rust/src/hud.rs

```diff
@@ -26,12 +26,17 @@ impl Hud {
     pub fn show_game_over(&self) {
         self.show_message("Game Over".into());
 
+        let mut timer = self.base.get_tree().unwrap().create_timer(2.0).unwrap();
+        timer.connect("timeout".into(), self.base.callable("_show_start_button"));
+    }
+
+    #[func]
+    fn _show_start_button(&mut self) {
         let mut message_label = self.base.get_node_as::<Label>("MessageLabel");
-        message_label.set_text("Dodge the\nCreeps!".into());
+        message_label.set_text("Dodge The Creeps".into());
         message_label.show();
-
         let mut button = self.base.get_node_as::<Button>("StartButton");
-        button.show();
+        button.show()
```

Comment:
Why remove the semicolon?

Reply:
It doesn't need to be removed. I'll put it back and squash into single commit.